backend/src/packages/users/user.model.ts

```diff
@@ -1,7 +1,11 @@
+import { Model } from 'objection';
+import { type RelationMappings } from 'objection';
```

Comment:
```suggestion
import { type RelationMappings, Model } from 'objection';
```

Reply:
sorry, i thought it was only in user details model